main.tf

```diff
@@ -135,6 +135,7 @@ module "aci_aaep" {
   physical_domains   = [for dom in lookup(each.value, "physical_domains", []) : "${dom}${local.defaults.apic.access_policies.physical_domains.name_suffix}"]
   routed_domains     = [for dom in lookup(each.value, "routed_domains", []) : "${dom}${local.defaults.apic.access_policies.routed_domains.name_suffix}"]
   vmware_vmm_domains = lookup(each.value, "vmware_vmm_domains", [])
+  endpoint_groups    = [for epg in lookup(each.value, "endpoint_groups", []) : "${epg}${local.defaults.apic.tenants.application_profiles.endpoint_groups.name_suffix}"]
```

Comment:
In this case it is not a list of EPG names, but rather a list of dicts with various attributes (tenant, ap, epg, etc.). So this should be:
```
endpoint_groups = lookup(each.value, "endpoint_groups", [])
```

Reply:
this matches what you are saying now.